Automattic/jetpack
# Publicize: Remove Path UI

Partly fixes #10155.
Still needs Calypso ( https://github.com/Automattic/wp-calypso/issues/27252 ) and backend.

#### Changes proposed in this Pull Request:

* Remove Path UI

#### Testing instructions:

* Visit Settings->Sharing in wp-admin with Publicize activated.
* Confirm Path is no longer listed


#### Proposed changelog entry for your changes:

* Removes Path as an option for Publicize since that service is closing October 2018.
